pages/payment.js

```diff
@@ -0,0 +1,281 @@
+import Head from 'next/head';
+import Header from '../components/Header.js';
+import Footer from '../components/Footer.js';
+import Link from 'next/link';
+import cookie from 'js-cookie';
+import React from 'react';
+import nextCookies from 'next-cookies';
```

Comment:
This could be imported in the `getServerSideProps` function so that it only adds to the code on the server (it's not used anywhere on the client-side).